### DISCUSSION TITLE
OSM tiles with place names in English

### DISCUSSION THREAD

**Discussion by StevenSarens** [osmdroid/osmdroid#1845] [Q&A]
I currently use 

` myMap.setTileSource(TileSourceFactory.MAPNIK);`

to download OSM tiles for the test version of my app. That gives the general map, with place names written in the local language and local alphabet. What code do I use to download tiles with all place names in English, worldwide?

**Comment by monsieurtanuki** [comment]
Hi @StevenSarens!
I think I understand what you mean but I don't know which source you should use instead (here it's a matter of tile source).
I've just had a look at https://wiki.openstreetmap.org/wiki/Tile_servers and could not find a free tile source with 100% English names. You may have a look at https://www.maptilesapi.com and their "free basic plan for non-commercial hobby projects and internal development testing".

**Comment by Psijic** [comment]
The best way to localize is to use raster tiles with vector labels and POIs. A lot of maps support it, maybe osmdroid is ready for it?
